### PR TITLE
Issue #56 Show something when the game is on pause

### PR DESCRIPTION
**Pull request by Moloss**
Text and background have been implemented during the pause of the game.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -59,4 +58,15 @@ body {
...
+}
+.pause {
+    z-index: 3;
+    position: absolute;
+    padding-top: 20%;
+    width: 100%;
+    height: 100%;
+    background-color: black;
+    opacity: 0.8;
+    filter:alpha(opacity=80); /* For IE <=8 */
```
Comment (Yoric) [inline css/style.css:70]: The game is probably not going to work with IE <= 8, but ok.

File: index.html [hunk 2]
```diff
@@ -30,5 +30,10 @@
 <audio src="audio/music.ogg" autoplay="autoplay" controls="" loop="" style="display: none;"></audio>
 -->
 </div>
+<div class="pause hidden" id="pause">
+<hr />
+<p>The game is paused. <br />Press space to continue...</p>
```
Comment (Yoric) [inline index.html]: Just write `Pause` and let people figure out how to unpause.

**Comment by Yoric** [comment]
Looks almost good. Could you please make the changes I requested?

Also:
- could you configure your github account to have personal pages? This will ensure that I can test your fork just by going to
  http://moloss.github.com/piranhas
- could you add your name to file `AUTHORS` in your patch?

**Comment by Moloss** [comment]
Now, it's good.

**Pull request merged**